java8-okhttp-gson/src/example/java/com/sphereon/sdk/pdf/stamper/StampPdf.java

```diff
@@ -147,33 +164,59 @@ public static StamperConfigResponse updateConfiguration(final String configId, f
         signatureComponent.setSignatureMode(SignatureModeEnum.CERTIFICATION);
         config.setSignatureComponent(signatureComponent);
 
-        // update the configuration with the new settings
+        // Update the configuration with the new settings.
         return configApi.updateConfiguration(configId, config);
     }
 
+    private static void createCryptoKeysConfiguration() throws IOException {
+        // Create connection.
+        final URL url = new URL(cryptoKeysApiUrl + "/manage/configs");
+        final HttpURLConnection connection = (HttpURLConnection) url.openConnection();
+        connection.setRequestMethod("POST");
+        connection.setRequestProperty("Content-Type", "application/json; utf-8");
+        connection.setRequestProperty("Accept", "application/json");
+        connection.setRequestProperty("Authorization", "Bearer " + accessToken);
+        connection.setDoInput(true);
+        connection.setDoOutput(true);
+
+        // Create body (JSON).
+        final String requestBody = "{\"configuration\": {\"name\": " + cryptoKeysConfigName + ",\"azureKeyVaultSettings\": {\"tenant\": " + azureTenant + ",\"clientId\": "+ azureClientId + ",\"clientSecret\":" + azureClientSecret + ",\"resourceGroup\": \"certificates\",\"keyVaultName\": \"sphereon-certs\",\"keyVaultURL\": \"https:\\/\\/sphereon-certs.vault.azure.net\\/\",\"environment\": \"AZURE\",\"region\": \"GERMANY_CENTRAL\",\"subscriptionId\": " + azureSubscriptionId + ",\"hsmUsage\": null\\},\"localStorageSettings\": null,\"implementationType\": \"AZURE_KEYVAULT_MANAGED\",\"storageTypeType\": \"AZURE_KEYVAULT\\}";
```

Comment:
If we can format it better or use some object to construct it, that will be better. Currrently the JSon does not seem to be valid. Causing 400 and hence test is not passing.

`// Create body (JSON).
        final String requestBody = "{" +
            "\"configuration\": {" +
                "\"name\": \"" + cryptoKeysConfigName + "\"," +
                "\"azureKeyVaultSettings\": {" +
                    "\"tenant\": \"" + azureTenant + "\"," +
                    "\"clientId\": \""+ azureClientId + "\"," +
                    "\"clientSecret\": \"" + azureClientSecret + "\"," +
                    "\"resourceGroup\": \"certificates\"," +
                    "\"keyVaultName\": \"sphereon-certs\"," +
                    "\"keyVaultURL\": \"https:\\/\\/sphereon-certs.vault.azure.net\\/\"," +
                    "\"environment\": \"AZURE\"," +
                    "\"region\": \"GERMANY_CENTRAL\"," +
                    "\"subscriptionId\": \"" + azureSubscriptionId + "\"," +
                    "\"hsmUsage\": null" +
                "}," +
                "\"localStorageSettings\": null," +
                "\"implementationType\": \"AZURE_KEYVAULT_MANAGED\"," +
                "\"storageTypeType\": \"AZURE_KEYVAULT\"" +
            "}" +
        "}";`

Reply:
Also 

```
{
  "errors": [
    {
      "level": "FATAL",
      "code": "exception",
      "message": "Name already exists: sphereon-certs",
      "cause": null
    }
  ]
}
```

Reply:
Agree. You could add a crypto-keys-sdk as test dep for the object route

Reply:
Fixed the JSON, seems the escaping was incorrect. And for now i just extended the error handling part. So if we get the 400 with Name already exists we just continue